src/pages/library/WorkflowsLibrary.tsx

```diff
@@ -0,0 +1,232 @@
+          position: 'relative',
+        }}
+      >
+        {_.map((wf: CuratedWorkflowDetails) => {
+          return (
+            <Clickable
+              key={wf.key}
+              href={wf.url}
+              {...Utils.newTabLinkProps}
+              style={{
+                ...Style.elements.card.container,
+                width: 150,
+                height: 110,
+                margin: '0 1rem 2rem 0',
+                color: colors.accent(1),
+              }}
+            >
+              <div>{wf.title}</div>
+              <div style={{ bottom: 0 }}>{icon('pop-out', { size: 14, style: { marginLeft: '0.25rem' } })}</div>
+            </Clickable>
+          );
+        }, curatedWorkflowsList(dockstoreUrlRoot))}
+      </div>
+    </div>
+  );
+};
+
+export const WorkflowsLibrary = () => {
+  const dockstoreUrl = `${getConfig().dockstoreUrlRoot}/search?_type=workflow&descriptorType=WDL&searchMode=files`;
+  const workflowsRepoUrl: string = isFeaturePreviewEnabled(FIRECLOUD_UI_MIGRATION)
+    ? Nav.getLink('workflows')
+    : `${getConfig().firecloudUrlRoot}/?return=${getEnabledBrand().queryName}#methods`;
+
+  return (
+    <FooterWrapper alwaysShow>
+      {libraryTopMatter('workflows')}
+      <div role='main' style={{ flexGrow: 1 }}>
+        <div style={{ display: 'flex', flexDirection: 'row', height: '100%', flex: 1 }}>
+          <div style={{ width: '80%', display: 'flex' }}>
+            <div style={{ flex: 1, margin: '30px 0 30px 40px' }}>
+              <div style={styles.header}>Discover Workflows</div>
+              <div style={{ display: 'flex', marginTop: '0.25rem' }}>
+                <div>
+                  <WorkflowSourceBox
+                    title='Dockstore.org'
+                    description='A community repository of public workflows that offers automatic integration with GitHub and publishing features.'
+                    url={dockstoreUrl}
+                    logoFilePath={dockstoreLogo}
+                  />
+                </div>
+                <div style={{ marginLeft: 20 }}>
+                  <WorkflowSourceBox
+                    title='Broad Methods Repository'
+                    description='A repository of WDL workflows that offers quick hosting of public and private workflows.'
+                    url={workflowsRepoUrl}
+                    logoFilePath={terraLogo}
+                  />
+                </div>
+              </div>
+              <CuratedWorkflowsSection />
+            </div>
+          </div>
+          <div
+            style={{
+              width: '20%',
+              padding: '25px 30px',
+              backgroundColor: colors.light(0.7),
+              lineHeight: '20px',
+            }}
+          >
+            <div style={{ fontSize: '1rem', fontWeight: 600 }}>Helpful Workflow support links</div>
+            <p>Learn how to configure your workflow analysis, including options to save time and money.</p>
+            <div>
+              <ul>
```

Comment:
Note to reviewers: For this right side support links panel, the mock-up contains more than 3 links. But after talking with Beth it was decided that 10 links would be too many and hence these 3 were decided to be put in the panel.